haas/test_common.py

```diff
@@ -75,11 +77,16 @@ def database_only(f):
     def config_initialize():
         # Use the 'null' backend for these tests
         cfg.add_section('general')
-        cfg.set('general', 'driver', 'null')
+        cfg.set('general', 'driver', 'null_vlan')
         cfg.add_section('devel')
         cfg.set('devel', 'dry_run', True)
         cfg.add_section('headnode')
         cfg.set('headnode', 'base_imgs', 'base-headnode, img1, img2, img3, img4')
+	cfg.add_section('database')
+	cfg.set('database', 'sqlite', 'sqlite:///:memory:')
+	cfg.set('database', 'postgres', 'postgresql://postgres:postgres@localhost/haas')
```

Comment:
You've got these two lines in several places. This is analogous to:

     x = 1
     x = 2

...which is silly for obvious reasons.